relay-event-normalization/src/event.rs

```diff
@@ -614,32 +626,36 @@ fn normalize_device_class(event: &mut Event) {
 /// Process the last frame of the stacktrace of the first exception.
 ///
 /// No additional frames/stacktraces are normalized as they aren't required for metric extraction.
-fn normalize_stacktraces(event: &mut Event) -> ProcessingResult {
+fn normalize_stacktraces(event: &mut Event) {
```

Comment:
We could simplify this function with `get_value!`, but that's out of scope for this PR.